nautilus_trader/test_kit/stubs/persistence.py

```diff
@@ -77,6 +77,9 @@ def schema():
     @staticmethod
     def news_events() -> list[NewsEventData]:
         df = pd.read_csv(TEST_DATA_DIR / "news_events.csv")
+        # Use only first 5000 rows for faster testing (vs original 86,985 rows)
```

Comment:
We can probably move these explanatory comments into the PR description (for the next one).